### DISCUSSION TITLE
Rerun button not available in 2.6.2 airflow for each task instance

### DISCUSSION THREAD

**Discussion by DivyasriDaka** [apache/airflow#32388] [Q&A]
Rerun button not available in 2.6.2 airflow for each task instance, how can we resolve this?

We need rerun/run option if there are any failures or usecase to run specific task

**Comment by potiuk** [comment]
The Run option has been brokken and it could have worked accidentally for some uers but we had to remove it because it have been really not doing what it was supposed to do (i.e. tasks - were attempted to run from the webserver, and in Airlfow 2 - webserver should not even have the possibility of accessing DAG files). You should use Clear functionality instead.

More details: https://github.com/apache/airflow/pull/29706 introduced in 2.6.0

**Reply by notatallshaw** [reply]
Finally had time to upgrade airflow and immediately hit this, clear functionality does not work well for my use case of the keeping the DAG paused when I force run a few tasks. 😞
